code/game/atoms.dm

```diff
@@ -493,6 +493,7 @@
 	if(istype(blood_DNA, /list))
 		blood_DNA = null
 		return TRUE
+	blood_color = null //chompfixy, cleaning objects saved its future blood color no matter what
```

Comment:
Should be tested with the robolimb datums. They all have custom blood_color values defined.

Reply:
Should be fine, they are independent.